types/usb/index.d.ts

```diff
@@ -276,9 +280,9 @@ export class Interface {
    * It is an error to release an interface with pending transfers.
    *
    * The device must be open to use this method.
-   * @param cb
+   * @param callback
    */
-  release(cb?: (err?: string) => void): void;
+  release(callback?: (error?: LibUSBException) => void): void;
```

Comment:
Are we sure all returned errors are `LibUSBException`?

Reply:
Yes, I'm reasonably sure. I reviewed the 1.5.0 code, and:
 - In the JS side, no errors are generated, all of them come from the native side.
 - In the native side, the only place I found where other errors are thrown is [this one](https://github.com/tessel/node-usb/blob/cb760d799a0558c5160ba07874aceebcde495eaf/src/transfer.cc#L51-L61)
 - However, the JS side and typings make sure the device is open (you wouldn't be able to access endpoints otherwise), that arg[0] is a Buffer, and that the transfer is submitted only once. Except for edge cases (using internals and such), it should be safe to assume callback errors are `LibUSBException`.

Reply:
Convinced me :)